### PR TITLE
indicate file name requried for static file

### PR DESCRIPTION
**Pull request by rwoodnz**
the text doesn't say explicitly to call your file comments.json, nor to put it in the public directory.

### CODE REVIEW

File: docs/docs/tutorial.md [hunk 1]
```diff
@@ -374,7 +374,7 @@ var CommentBox = React.createClass({
 `getInitialState()` executes exactly once during the lifecycle of the component and sets up the initial state of the component.
 
 #### Updating state
-When the component is first created, we want to GET some JSON from the server and update the state to reflect the latest data. In a real application this would be a dynamic endpoint, but for this example, we will use a static JSON file to keep things simple:
+When the component is first created, we want to GET some JSON from the server and update the state to reflect the latest data. In a real application this would be a dynamic endpoint, but for this example we will keep things simple by creating a static JSON file public/comments.json containing the array of comments:
 
 ```javascript
 // tutorial13.json
```
Comment (jimfb) [inline docs/docs/tutorial.md]: It seems weird to call it `tutorial13.json` if everyone expects it to be `comments.json`.  @spicyj @zpao, is there a reason this comment doesn't say `comments.json`.

File: docs/docs/tutorial.md [hunk 2]
```diff
@@ -374,7 +374,7 @@ var CommentBox = React.createClass({
 `getInitialState()` executes exactly once during the lifecycle of the component and sets up the initial state of the component.
 
 #### Updating state
-When the component is first created, we want to GET some JSON from the server and update the state to reflect the latest data. In a real application this would be a dynamic endpoint, but for this example, we will use a static JSON file to keep things simple:
+When the component is first created, we want to GET some JSON from the server and update the state to reflect the latest data. In a real application this would be a dynamic endpoint, but for this example we will keep things simple by creating a static JSON file public/comments.json containing the array of comments:
```
Comment (jimfb) [inline docs/docs/tutorial.md]: Let's put public/comments.json in backticks, so it becomes: `public/comments.json`, as done for `getInitialState` in the line above.

**Comment by jimfb** [comment]
Seems reasonable to me.

Also, @rwoodnz can you sign the CLA so we can merge?  You know, lawyers :/

**Comment by rwoodnz** [comment]
@jimfb. Cheers, yep I did the CLA. If I go to it now I get an error saying my email has already been used for it.

**Comment by jimfb** [comment]
RE CLA: Well, that's annoying.  I found your record in the CLA database, but there is no github account attached to the record.  I'll try to figure out the CLA situation on our end and ping you when I've figured out next steps.

**Comment by rwoodnz** [comment]
Yep. My apologies on that one. Rushed through it and hit agree without filling in the box. After that you can't do it again.

File: docs/docs/tutorial.md (hunk 1 above)
Comment (zpao) [inline docs/docs/tutorial.md]: The tutorial is broken into steps with file names to match (eg `tutorial13.js` would not actually be named that on disk, just used to indicate the step of the tutorial). Honestly we could probably get rid of those comments entirely.

**Comment by jimfb** [comment]
Ok, CLA situation has been solved.  @rwoodnz, if you can amend your commit to put the public/comments.json in backticks (the key just above the escape key, \`, so it becomes `public/comments.json`), I think we'll be ready to merge.

**Comment by zpao** [comment]
:thumbsup:

**Comment by fredericksilva** [comment]
:+1:

**Pull request merged**